src/main/java/software/sigma/bu003/internship/vedmid_andrii/controller/PartController.java

```diff
@@ -0,0 +1,54 @@
+package software.sigma.bu003.internship.vedmid_andrii.controller;
+
+import lombok.AllArgsConstructor;
+import org.modelmapper.ModelMapper;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import software.sigma.bu003.internship.vedmid_andrii.dto.PartDTO;
+import software.sigma.bu003.internship.vedmid_andrii.entity.Part;
+import software.sigma.bu003.internship.vedmid_andrii.service.PartService;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/part")
+@AllArgsConstructor
+public class PartController {
+
+    private final PartService partService;
+
+    private final ModelMapper modelMapper = new ModelMapper();
+
+    @PostMapping
+    public ResponseEntity<Part> createPart(@RequestBody PartDTO partDTO) {
```

Comment:
you can return Part you don't need ResponseEntity.... the same for all the controllers, also I don't see validations of Part, there are ready to use tools in spring boot for this

Reply:
I am returning ResponseEntity because when I will add OpenApi with swagger and implement the interface from them, I will need to rewrite all the endpoints, return ResponseEntity